src/components/v2/Notification/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+export type NotificationType = 'cui-is-success' | 'cui-is-info' | 'cui-is-warning' | 'cui-is-danger';
+
+interface NotificationProps {
+    description: string;
+    title?: string;
+    hasIcon?: boolean;
+    type: NotificationType;
+    isOutlined?: boolean;
+    actionBtnHref?: string;
+    actionBtnText?: string;
+    onActionCb?: Function;
+    onCloseCb?: boolean;
```

Comment:
varför inte bara onClose?

Reply:
Det blev fel där hehe. Det är en funktion som ska in där som hanterar knapptryckningen